Provisional schedule (#1178)
writethedocs/www: docs/conf/portland/2020/news/schedule.rst

:template: {{year}}/generic.html

.. post:: July 17, 2020
   :tags: {{year}}, portland-{{year}}, schedule

Announcing Full Schedule
========================

Hey everyone!

We're less than a month away from the first ever online Portland conference and we're working hard to make sure it'll be as welcoming and as fun as the in-person event you're probably familar with.

Today we're announcing the full presentation schedule and the platform we'll be hosting the conference on.

> We are working on last minute details on a few more activities during the conference, so look for more that information in our next mailing.

Reviewer: ```suggestion
We are working on last-minute details for a few more activities during the conference, so look out for more information in our next mailing.
```

* **Lightning Talks.** We'll have `lightning talks <https://www.writethedocs.org/conf/portland/{{year}}/lightning-talks/>`__ each day after lunch. These are 5 minute presentations and a lot of fun. They can be about anything you want, this is your chance to share something you are passionate about with the audience. Slides are not required.
* **Unconference.** We'll be running the `Unconference <https://www.writethedocs.org/conf/portland/{{year}}/unconference/>`_ all day Monday and on Tuesday afternoon. Want to chat with like-minded folks about a specific thing? This is the place.
* **Job Fair**. As with the previous few years, on Tuesday morning we'll have a `job fair <https://www.writethedocs.org/conf/portland/{{year}}/job-fair>`_ where you can find your next documentarian friendly job.
* **Badge Flair Contest**. We're working on adapting this to the new format. Details will be posted as soon as we have them.

> * **Social event**. On Monday night the main stage will become the main social event space. We will post details about the program as soon as it's finalized. You're also welcome to create sessions for any board games, trivia quizzes, karaoke, or any other social activity that you want to share with your fellow documentarians.

Reviewer: ```suggestion
* **Social event**. On Monday night the conference platform will become the social event space. The main stage will be closed, the main focus will be attendee-created sessions. You're welcome to create sessions for board games, trivia quizzes, karaoke, or any other social activity that you want to share with your fellow documentarians.
```

Reviewer: Sorry I didn't update the actual event yet to reflect that there's no main stage anymore ;)

> * **Help Desk**. If you have any questions about navigating the virtual event, go to the Write the Docs Helpdesk booth and chat with one of the organizers. Our Welcome Wagon guide can also help you make the most out of the conference.

Reviewer: ```suggestion
* **Conference Help Desk**. If you have any questions about navigating the virtual event, we'll be running the Write the Docs Helpdesk booth, where you can chat with one of the organizers and the Welcome Wagon team.
```

:template: {{year}}/generic.html

.. post:: July 17, 2020
   :tags: {{year}}, portland-{{year}}, schedule

Announcing Full Schedule
========================

Hey everyone!

We're less than a month away from the first ever online Portland conference and we're working hard to make sure it'll be as welcoming and as fun as the in-person event you're probably familar with.

> Today we're announcing the full presentation schedule and the onnline platform we'll be hosting the conference on.

Reviewer 2: Party, badge flair, Welcome Wagon,  etc. 

```suggestion
Today we're announcing the full presentation schedule and the platform we'll be hosting the conference on.
We are working on last minute details on a few more activities during the conference, so look for more that information in our next mailing.
```

Conference platform
--------------------

> The conference will be held on https://hopin.to/, which not only provides a stage for our amazing speakers, but also has integrated chat, unconference rooms with breakout calls, and impromptu 1-on-1 meetings. We're updating our welcome guide so it'll all feel familar before you've even arrived.

Reviewer 2: ```suggestion
The conference will be held on https://hopin.to/, which not only provides a stage for our amazing speakers, but also has integrated chat, unconference rooms with breakout calls, and impromptu 1-on-1 meetings. We're updating our Welcome Wagon guide, to be released soon, so it'll all feel familiar before you've even arrived.
```

> The conference will be held on https://hopin.to/, which not only provides a stage for our amazing speakers, but also has integrated chat, unconference rooms with breakout calls, and impromptu 1-on-1 meetings. We're updating our welcome guide so it'll all feel familar before you've even arrived.

Reviewer: ```suggestion
The conference will be held on https://hopin.to/, which not only provides a stage for our amazing speakers, but also has integrated chat, unconference rooms with breakout calls, expo booths, and impromptu 1-on-1 meetings. We're updating our welcome guide so it'll all feel familiar before you've even arrived.
```

Full presentation schedule
--------------------------

> The conference website is now updated with the `talk schedule <https://www.writethedocs.org/conf/portland/{{year}}/schedule/>`_ for the main stage presentations, so you can start planning your time (and getting excited!) for the conference. We're excited to have Q&A after most talks

Reviewer 2: ```suggestion
The conference website is now updated with the `talk schedule <https://www.writethedocs.org/conf/portland/{{year}}/schedule/>`_ for the main stage presentations, so you can start planning your time (and getting excited!) for the conference. We're excited to have live Q&A after most talks, which will come from your questions but be moderated by an MC.
```

> As always, we'll be running the `unconference <https://www.writethedocs.org/conf/portland/{{year}}/unconference/>`_, and `lightning talks <https://www.writethedocs.org/conf/portland/2019/lightning-talks/>`_ each day after lunch.

Reviewer 2: We should either cut this section since we mention all this in detail below, or add the job fair here.

Reviewer: I think it's best to remove both of these links, since we're listing job fair and everything else later in the post.

> As always, we'll be running the `unconference <https://www.writethedocs.org/conf/portland/{{year}}/unconference/>`_, and `lightning talks <https://www.writethedocs.org/conf/portland/2019/lightning-talks/>`_ each day after lunch.

Reviewer: ```suggestion
As always, we'll be running the `unconference <https://www.writethedocs.org/conf/portland/{{year}}/unconference/>`_, and `lightning talks <https://www.writethedocs.org/conf/portland/{{year}}/lightning-talks/>`_ each day after lunch.
```

You'll be able to sign up for unconference sessions and lightning talks during the conference.

Start thinking about Writing Day
--------------------------------

> Our pre-conference `Writing Day <https://www.writethedocs.org/conf/portland/2019/writing-day/>`_ will happen on Sunday, and it is much more fun when folks come with their own projects in mind!

Reviewer: ```suggestion
Our pre-conference `Writing Day <https://www.writethedocs.org/conf/portland/{{year}}/writing-day/>`_ will happen on Sunday, and it is much more fun when folks come with their own projects in mind!
```

If you have a project or are part of a community that you think would benefit from a writing sprint, start brainstorming now about what you might want to focus on.
If you'd like us to include your project on the Writing Day page, drop us an email at `portland@writethedocs.org <mailto:portland@writethedocs.org>`_.
Or just show up on Sunday ready to introduce your project!

Shirts on sale
--------------

> The Write the Docs Portland 2019 Pop-Up Shop is now open! We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want!

Reviewer 2: Link?

```suggestion
The `Write the Docs Portland 2020 Pop-Up Shop <https://teespring.com/wtd-portland-2020>`_ is still open! We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want. 
```

> The campaign will run in 3-day batches until TODO.

Reviewer: ```suggestion
The campaign will run in 3-day batches until August 30th. Shipping is available from the USA or Europe. However, due to the global shipping situation, we recommend you order your shirt at least 2-3 weeks in advance if you want to sport your new threads during the conference.
```

Author: Got a link?

Reviewer: Damn, I thought I added it to the suggestion. Here it is again: https://teespring.com/wtd-portland-2020

> Other events and activities

Reviewer 2: ```suggestion
All our events and activities
```

----------------------------

Write the Docs is more than a conference. Every year we make sure to have a variety of activities that our attendees can enjoy, whether you want to learn something, meet new people, or share knowledge. And even though this years conference is held online, we're holding to that plan.

* **Lightning Talks.** We'll have `lightning talks <https://www.writethedocs.org/conf/portland/{{year}}/lightning-talks/>`__ each day after lunch. These are 5 minute presentations and a lot of fun. They can be about anything you want, this is your chance to share something you are passionate about with the audience. Slides are not required.
* **Unconference.** We'll be running the `Unconference <https://www.writethedocs.org/conf/portland/{{year}}/unconference/>`_ all day Monday and on Tuesday afternoon. Want to chat with like-minded folks about a specific thing? This is the place.

> * **Job Fair**. Following last year's great success, on Tuesday morning we'll have a `job fair <https://www.writethedocs.org/conf/portland/{{year}}/job-fair>`_ where you can find your next documentarian friendly job.

Reviewer 2: ```suggestion
* **Job Fair**. As with the previous few years, on Tuesday morning we'll have a `job fair <https://www.writethedocs.org/conf/portland/{{year}}/job-fair>`_ where you can find your next documentarian friendly job.
```

> * **Job Fair**. Following last year's great success, on Tuesday morning we'll have a `job fair <https://www.writethedocs.org/conf/portland/{{year}}/job-fair>`_ where you can find your next documentarian friendly job.

Reviewer: ```suggestion
* **Job Fair**. On Tuesday morning we'll have a `job fair <https://www.writethedocs.org/conf/portland/{{year}}/job-fair>`_ where you can find your next documentarian-friendly job.
```

> 

Reviewer: Here we should add something about the social event, badge contest, and the helpdesk maybe?

Author: Wanna suggest some copy? <3 @thatdocslady
